scripts/run_ganache.sh

```diff
@@ -10,7 +10,7 @@ ganache_running() {
 
 start_ganache() {
     if [ "$SOLIDITY_COVERAGE" = true ]; then
-        node_modules/.bin/testrpc-sc -l 0xfffffffffff -p "$ganache_port" -a 100 > /dev/null &
+        node_modules/.bin/testrpc-sc -k "petersburg" -l 0xfffffffffff -p "$ganache_port" -a 100 > /dev/null &
```

Comment:
Petersburg should be the default right now given the version of `solidity-coverage`/`testrpc-sc` used so we can omit this flag.